Control/Monad/STM.hs

```diff
@@ -105,7 +105,8 @@ catchSTM (STM m) handler = STM $ catchSTM# m handler'
 -- | A variant of 'throw' that can only be used within the 'STM' monad.
 --
 -- Throwing an exception in @STM@ aborts the transaction and propagates the
--- exception.
+-- exception. (Note: Allocation effects, such as  'newTVar' are not roll backed
```

Comment:
"Rolled back", rather than "roll backed"

Reply:
My bad, it is fixed now!